--- gkrellm-radio-0.2.1/Makefile~	Sun Feb 25 14:42:09 2001
+++ gkrellm-radio-0.2.1/Makefile	Mon Sep  3 18:58:13 2001
@@ -12,7 +12,7 @@
 CC=gcc
 LDFLAGS=
 OBJS=gkrellm_radio.o radio.o
-CFLAGS=-fPIC -I$(GKRELLMDIR)/include `gtk-config --cflags` `imlib-config --cflags-gdk` -Wall -DVERSION=\"$(VERSION)\"
+CFLAGS=$(OPT) -fPIC -I$(GKRELLMDIR)/include `gtk-config --cflags` `imlib-config --cflags-gdk` -Wall -DVERSION=\"$(VERSION)\"
 
 radio.so: $(OBJS)
 	$(CC) -shared -Wl -o radio.so $(OBJS)
